metricflow/aggregation_properties.py

```diff
@@ -2,7 +2,7 @@
 
 from enum import Enum
 
-from dbt_semantic_interfaces.objects.aggregation_type import AggregationType
+from dbt_semantic_interfaces.type_enums.aggregation_type import AggregationType
```

Comment:
@QMalcolm Mind verifying the package name change?

Reply:
Looks right. https://github.com/dbt-labs/dbt-semantic-interfaces/pull/33/